pkg/omf/functions/util/omf.check.version.fish

```diff
@@ -0,0 +1,3 @@
+function omf.check.version -a min_version -a current_version
+  test (echo "$min_version"\n"$current_version" | sort -V | head -n1) = "$min_version"
```

Comment:
the current chunk count thing was because i couldn't figure out a way to make the array stuff happy when comparing, for example, min version 1.2.3 with current version 1.2.

any better ideas?